consensus/src/context.rs

```diff
@@ -248,8 +253,22 @@ pub enum BlockChainContextRequest {
         /// This will panic if the number of blocks will pop the genesis block.
         numb_blocks: usize,
     },
+
+    /// Get information on a certain hardfork.
+    HardForkInfo(HardFork),
+
+    /// Get the current fee estimate.
+    FeeEstimate {
+        /// TODO
+        grace_blocks: u64,
+    },
+
     /// Clear the alt chain context caches.
     ClearAltCache,
+
+    /// Get information on all the current alternate chains.
+    AltChains,
```

Comment:
### Signatures (blockchain context request)
Added to the existing `BlockChainContextRequest`.